### PR TITLE
Remove app-templates for EOL versions

### PR DESCRIPTION
**Pull request by provokateurin**
I split it up because the rename detection was going a bit wrong (ok Github still shows it wrong if you look at the overall diff).

### CODE REVIEW

**Comment by bigcat88** [comment]
the template from 26 version should not be touched, since the templates for 27 and 28 are symlinks for it

**Comment by provokateurin** [comment]
I didn't touch it, I just replace the 27 template with the one for 26

File: nextcloudappstore/scaffolding/tests/data/info.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <category>tools</category>
     <bugs>https://test.com</bugs>
     <dependencies>
-        <nextcloud min-version="24" max-version="24"/>
+        <nextcloud min-version="28" max-version="28"/>
```
Comment (provokateurin) [inline nextcloudappstore/scaffolding/tests/data/info.xml:19]: Shouldn't the min version be 27 because the template is also for 27?

Comment (bigcat88) [reply nextcloudappstore/scaffolding/tests/data/info.xml:19]: tests were modified to test against 28 version, to not modify them in 3 months again..

**Review by bigcat88 — APPROVED** [review]

**Pull request merged**